<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CuteNodes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="NodeView" name="nodeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewScene"/>
    <addaction name="actionLoadScene"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScene"/>
    <addaction name="actionSaveSceneAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionToggleGrid"/>
    <addaction name="actionToggleSnap"/>
    <addaction name="actionToggleOverlap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewScene"/>
   <addaction name="actionLoadScene"/>
   <addaction name="actionSaveScene"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionToggleGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid &amp;Visibility</string>
   </property>
   <property name="toolTip">
    <string>Toggle grid visibility</string>
   </property>
  </action>
  <action name="actionToggleSnap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid &amp;Snapping</string>
   </property>
   <property name="toolTip">
    <string>Toggle grid snapping</string>
   </property>
  </action>
  <action name="actionToggleOverlap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Node &amp;Overlapping</string>
   </property>
   <property name="toolTip">
    <string>Toggle Node overlap</string>
   </property>
  </action>
  <action name="actionLoadScene">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/load_scene.png</normaloff>:/icons/load_scene.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Scene</string>
   </property>
   <property name="toolTip">
    <string>Load scene from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/save_scene.png</normaloff>:/icons/save_scene.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Scene</string>
   </property>
   <property name="toolTip">
    <string>Save scene to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveSceneAs">
   <property name="text">
    <string>Save Scene &amp;As ...</string>
   </property>
   <property name="toolTip">
    <string>Save Scene to a different file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNewScene">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/new_scene.png</normaloff>:/icons/new_scene.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Scene</string>
   </property>
   <property name="toolTip">
    <string>Create new empty scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NodeView</class>
   <extends>QGraphicsView</extends>
   <header>NodeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
